playwright/_impl/_browser_context.py

```diff
@@ -299,3 +372,7 @@ async def new_cdp_session(self, page: Page) -> CDPSession:
         return from_channel(
             await self._channel.send("newCDPSession", {"page": page._channel})
         )
+
+    @property
+    def tracing(self) -> Tracing:
```

Comment:
I don't think we are ready for the tracing just yet, we are still working on the API shape. Can we roll without tracing?

Reply:
discussed offline, rolling with it and maybe reverting / changing it then.

Reply:
@dgozman @mxschmitt fyi: Tracing was implemented in playwright-java too https://github.com/microsoft/playwright-java/pull/444